buildstockbatch/aws/aws.py

```diff
@@ -1509,10 +1509,6 @@ def validate_project(project_file):
     def docker_image(self):
         return "nrel/buildstockbatch"
 
-    @property
```

Comment:
This was moved to the base class [here](https://github.com/rewiringamerica/buildstockbatch/blob/427edcab4c7ca851934a91bb9b183dd2591cb687/buildstockbatch/cloud/docker_base.py#L126).